server/jetstream_cluster.go

```diff
@@ -4143,15 +4143,18 @@ func (js *jetStream) clusterInfo(rg *raftGroup) *ClusterInfo {
 
 	if rg == nil || rg.node == nil {
 		return &ClusterInfo{
-			Name:   s.ClusterName(),
-			Leader: s.Name(),
+			Name:     s.ClusterName(),
+			Leader:   s.Name(),
```

Comment:
Maybe make sub struct?

```go
Leader {
 Name: string
 ID: string
}```

Reply:
Then I'd rather just make it a `*PeerInfo`? there would be overlap then

Reply:
Pushed a commit doing that, it's what I wanted from the beginning :) but you didnt like the Current and Active in there that doesnt really mean much for a Leader.  Seems better though than adding a Leader struct thats only a few fields different from PeerInfo

Reply:
> I might suggest just going back to server name and use the audit tool etc.

We don't really know what the roadmap/plan is with audit tool, but we do need to ship peer membership before then